### ISSUE TITLE
leaking worlds after I updated and added dynamics

### ISSUE THREAD

**Issue by TLOU15** [CoFH/Feedback#432]
Hello, I have a crappy computer so I put together a few mods myself as it can not handle most packs.Recently I got frustrated trying to set up a automated system with extrautilities transfer nodes and filters and decided to update thermal expansion/foundation, CoFH and added Thermal Dynamics. I removed all old copies of the mods, deleted the config file and even started MC up and quit to make sure the .jar was rewritten without it. I added all the new versions and tried to play and I kept getting out of memory errors and crashes within a few minutes. Looking at my log it seems there is a leak some where? I am sorry it is such a short snip, pastebin wouldn't let me put in the whole thing so I put in what seemed the most important part.

http://pastebin.com/gKVJVc7v

Edit I forgot that all my thaumcraft nodes have a strange render to them now, they are flat squares layered on top of each other instead of circular shapes

**Comment by KingLemming** [comment]
Sounds like it may be a Thaumcraft thing, or something completely unrelated. We haven't seen any instances of a memory leak in any of our mods, but we'll keep an eye out.

**Comment by Entoarox** [comment]
Did you adjust the memory assignment for your instance to take into account the extra mods? You should have at least 2Gb assigned for any version of modded minecraft to work reliably, more if you have a sufficiently sized mod-set.

**Comment by TLOU15** [comment]
Sorry it has taken me so long to get back to you, I discovered that it was a derp on my behalf, I forgot to update my forge install. As I could only include part of the log you couldn't see the version of it. Once I updated to forge 1352 (I have enderstorage installed and I saw the solved issue with it), the leaking reports stopped. I still have a few lag issues but those are probably from my crappy computer. As for the rendering- I have included a image of my thaum node for you to see and will post it to the Azanor/thaumcraft github as well. 
 
I do like the label you assigned this.

Entoarox, sadly, as I said I have a crappy computer. An old 32 bit so the most I can give minecraft is 1 gb or it crashes before it starts up and tells me it can not create the dump file(?) I believe it is.This is why I put together my own mini pack as bare as I can. 

Thank you both for your time and considerable effort in making these wonderful mods I have been playing with TE since it first came out and it is one of my favorites.
 
![2015-04-15_08 53 24](https://cloud.githubusercontent.com/assets/11955707/7196152/77128e82-e481-11e4-8a4e-28af6263822b.png)

**Comment by Azanor** [comment]
Do you have optifine installed?

**Comment by TLOU15** [comment]
No sir I have had too many conflicts with optifine for the minimal FPS increase it gives. My computer is old and I am pushing it rather hard with the mods I have running most of the time I have it running on peaceful as the amount of entities from having mobs on tends to make MC lag and crash, I tried to make sure everything was up to date when I decided to add thermal dynamics *blush* except forge but I fixed that.  

I do have installed: 
Forge 1352
antiqueatlas-4.0.1a-1.7.10, 
Baubles- 1.7.10-1..3.1-7, 
BloodMagic- 1.7.10- 1.3.1-7, 
Botania r1.6-176, 
ChickenChunks-1.7.10-1.3.4.16-universal, 
CodeChickenCore-1.7.10-1.0.4.29-universal, 
CoFHCore-[1.7.10]3.0.2-262, 
EnderStorage-1.7.10-1.4.5.27-universal, 
extrautilities-1.2.1, 
ForgeMultipart-1.7.10-1.1.2.331-universal, 
InventoryTweaks-1.58-147, 
ironchest-1.7.10-6.0.62.742-universal, 
Jabba-1.2.0a_1.7.10, 
Mantle-1.7.10-0.3.2, 
natura-1.7.10-2.2.0.1, 
NotEnoughItems-1.7.10-1.0.4.83-universal,  
OpenBlocks-1.7.10-1.3, 
OpenModsLib-1.7.10-0.6, 
SoulShards-TOW-1.7.10-0.1-RC9-7, 
TConstruct-1.7.10-1.8.2a, 
Thaumcraft-1.7.10-4.2.3.5, 
ThermalDynamics-[1.7.10]1.0.0-122, 
ThermalExpansion-[1.7.10]4.0.1-182, 
ThermalFoundation-[1.7.10]1.0.0-81, 
Waila-1.5.8a_1.7.10

**Comment by Entoarox** [comment]
Did you try adding FastCraft? I think you might simply not have enough memory available, thus the textures for nodes are not getting loaded properly, FastCraft might be able to decrease memory usage just enough to circumvent the problem, but you might want to try removing another mod to bring your memory usage below the limit.

**Comment by TLOU15** [comment]
:D Entoarox I do it very very carefully, peaceful, limited exploring, no redstone clocks, minimal hoppers, render distance about 6 and no continuously running major systems.  I did add fastcraft as you suggested but it seems to not have made a difference I can see. 

I think I isolated the issue. I removed and re-added mods til the render change stopped happening and that was with Botania. I have posted to Vazkii. I want to thank you all for your assistance I deeply appreciate you have taken the time to help me.
